test/unit/client-tests.js

```diff
@@ -1,3 +1,5 @@
+// remove strict declaration since this test does rewiring.
+// eslint-disable-next-line strict
```

Comment:
We can still enable strict mode.
6 tests sets the value of the `Client` read-only property `profileManager`, which is not needed (can be removed).

It was my bad when implementing execution profiles, the manager was initially set on `connect()`. The final implementation uses a read-only property instead.